tests/integration/iface_admin_state_test.py

```diff
@@ -38,3 +40,20 @@ def test_set_a_down_iface_down():
     netapplier.apply(copy.deepcopy(desired_state))
 
     assertlib.assert_state(desired_state)
+
+
+@pytest.mark.xfail(reason='Some ifaces cannot be removed', strict=True)
```

Comment:
Should there also be an xfail for virtual interfaces that cannot be set to down, because down equals absent for them?

Reply:
Yes, but note that it exposes a larger scope problem: The way we handle `down` is completely wrong.
Do we have a Jira issue on this?

Reply:
I do not think that there is a Jira issue.